duckencoder.py

```diff
@@ -34,148 +33,155 @@ def parseScriptLine(line, keyProp, langProp):
                 result = ""
 
                 # split line into command and arguments
-                instr = line.split(" ", 1)
-                instr[0] = instr[0].strip()
-                if len(instr) > 1:
-                        instr[1] = instr[1].strip()
+                cmd, args = line.split(" ", 1)
```

Comment:
At this point the partition approach has to be used `cmd, _, args = line.partition(" ")`, otherwise `line="CONTROL"` would fail (as every other key instruction, which allows to omit args).

Guess we are ready to merge, if this is changed.